api/V1/Recording.js

```diff
@@ -199,19 +199,31 @@ module.exports = (app, baseUrl) => {
         });
       }
 
+      // If query should be filtered by tagged or not or ignored.
+      var sqlLiteral = '';
+      var tagRequired = false;
+      if (where._tagged == true) {
```

Comment:
how does `where._tagged` get its value from?

Reply:
From the users query. The web UI will add _tagged = true or false or not add it depending on what radio button is selected.